### DISCUSSION TITLE
The `set-output` command is deprecated

### DISCUSSION THREAD

**Discussion by korosuke613** [github/gh-actions-importer#72] [Feedback] [answered]
Some of the syntax is replaced by the `set-output` command. However, the `set-output` command is deprecated and will be completely disabled on May 31, 2023. I want it to be written `>> $GITHUB_OUTPUT` at the time of conversion.

> Starting 1st June 2023 workflows using save-state or set-output commands via stdout will fail with an error.
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Examples of use:
- https://github.com/github/gh-actions-importer/blob/880d4c6e0bad4d8acf6260e3e45fd3549da07f4a/docs/circle_ci/Orbs/Node/InstallPackages.md?plain=1#L24
- https://github.com/github/gh-actions-importer/blob/880d4c6e0bad4d8acf6260e3e45fd3549da07f4a/docs/circle_ci/Orbs/Slack/Notify.md?plain=1#L58

**Comment by assyadh** [comment]
Hello @korosuke613. Thanks for reporting this

**Comment by assyadh** [comment, marked as answer]
We'll be working on updating the command and release this in an upcoming version.

**Reply by korosuke613** [reply]
Thanks!

**Comment by assyadh** [comment]
Hi @korosuke613 - This is fixed in the latest release. Please run `gh actions-importer update` to consume it.

**Reply by korosuke613** [reply]
Confirmed. Thanks for responding!
